videospace_pot/server/__init__.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
```

Comment:
please remove all code comments that say copyright Kitware. Videospace should have an ALv2 header, there is no need to include the copyright Kitware @smadha